### DISCUSSION TITLE
Domain is not applied after changing project name!

### DISCUSSION THREAD

**Discussion by yurim45** [vercel/community#3172] [Help]
### Summary

I changed the project name the other day and deployed it.
It hasn't been applied to my domain since then.

When I go to the domains tab, I don't see the domain I registered
So, I'm trying to register my domain name.
Another vercel account user said it was in use.. That domain is mine..

Maybe something went wrong while renaming the project.
How should I connect to the original domain?

### Steps to Reproduce

Change project name > distribution > Not applicable to domain > When I checked the domains tab, my domain was not found. > When I try to register the domain again, it says another account is in use.

![image](https://github.com/vercel/community/assets/63839302/5877a302-9c37-40a3-90af-075a4857e67a)

**Comment by smaeda-ks** [comment]
@yurim45 what's the domain name?

**Reply by yurim45** [reply]
oh! Sorry for the late reply, this is my domain!
april5.world
